setup.py

```diff
@@ -39,6 +39,7 @@
         "sphinx_gallery",
     ],
     "dask": ["dask[complete]"],
+    "ray": ["ray[complete]"],
```

Comment:
```suggestion
    "ray": ["ray"],
```
I don't think the extra package `complete` exists for Ray. Maybe minimal dependencies would be sufficient? https://docs.ray.io/en/latest/ray-overview/installation.html